criu/cr-restore.c

```diff
@@ -2192,6 +2192,29 @@ static int write_restored_pid(void)
 	return 0;
 }
 
+static void reap_zombies(pid_t child_pid) {
+	pid_t current_pid;
+	int current_status;
```

Comment:
You can drop the "current_" prefix, we don't have any other variables here. And you can move declarations in the loop.